source/simulator/stokes_matrix_free.cc

```diff
@@ -1606,6 +1675,128 @@ namespace aspect
         mg_matrices_Schur_complement[level].fill_cell_data (level_viscosity_tables[level],
                                                             sim.pressure_scaling);
       }
+
+
+
+
```

Comment:
could you remove a few of these newline here? I think one or two newlines should do the job of separating them visually.